src/chapters/typescript/exercises.rst

```diff
@@ -30,152 +59,193 @@ Part 1 - Declare Variables With Type
-     * ``name: string;``
-     * ``speedMph: number;``
+      * ``milesPerKilometer: number = 0.621;``
+      * ``name: string;``
+      * ``speedMph: number;``
 
-   * Constructor
+   b. Constructor
 
-     * ``name`` is the first parameter and it MUST be a string.
-     * ``speedMph`` is the second parameter and it MUST be a number.
-     * Sets the class properties using ``this.name`` and ``this.speedMph``.
+      * ``name`` is the first parameter and it MUST be a string.
+      * ``speedMph`` is the second parameter and it MUST be a number.
+      * Sets the class properties using ``this.name`` and ``this.speedMph``.
 
-#. Move the function ``getDaysToLocation``, defined in Part 3, into the ``Spacecraft`` class.
+#. Move the function ``getDaysToLocation``, defined in Part 3, into the
+   ``Spacecraft`` class.
 
-   * Update the function to reference the class properties ``this.milesPerKilometer`` and ``this.speedMph``.
+   a. Update the function to reference the class properties
+      ``this.milesPerKilometer`` and ``this.speedMph``.
 
 #. Create an instance of the ``Spacecraft`` class.
 
-   * ``let spaceShuttle = new Spacecraft('Determination', 17500);``
+   a. ``let spaceShuttle = new Spacecraft('Determination', 17500);``
 
 #. Print out the days to Mars.
 
-   * Use template literals, ``${spaceShuttle.getDaysToLocation(kilometersToMars)}`` and ``${spaceShuttle.name}``.
+   a. Use template literals,
+      ``${spaceShuttle.getDaysToLocation(kilometersToMars)}`` and
+      ``${spaceShuttle.name}``.
 
 #. Print out the days to the Moon.
 
-   * Use template literals, ``${spaceShuttle.getDaysToLocation(kilometersToTheMoon)}`` and ``${spaceShuttle.name}``.
+   a. Use template literals,
+      ``${spaceShuttle.getDaysToLocation(kilometersToTheMoon)}`` and
+      ``${spaceShuttle.name}``.
 
-Expected Output
+#. Use the terminal in VSCode to compile your ``.ts`` file, then run the
+   ``part4.js`` file.
 
-::
+**Expected Output**
 
-   Determination would take 332.67857142857144 days to get to Mars.
-   Determination would take 0.5683628571428571 days to get to the Moon.
+::
 
+   $ tsc part4.ts
+   $ node part4.js
+      Determination would take 332.67857142857144 days to get to Mars.
+      Determination would take 0.5683628571428571 days to get to the Moon.
 
 Part 5 - Export and Import the SpaceLocation Class
---------------------------------------------------
-1. Fork the `part 5 repl.it <https://repl.it/@launchcode/ts-exercises-part-5>`_.
-2. In repl.it, add a new file named ``SpaceLocation.ts``.
-3. Paste in the below code to ``SpaceLocation.ts``.
+---------------------------------------------------
+
+#. From the file tree in VSCode, open the ``part5.ts`` file.
+#. In repl.it, add a new file named ``SpaceLocation.ts``.
+#. Paste in the below code to ``SpaceLocation.ts``.
 
-   * Notice the ``export`` keyword. That is what allows us to import it later.
+   a. Notice the ``export`` keyword. That is what allows us to import it later.
```

Comment:
For screen readers, the suggested practice is to use ol over ul (letters or numbers rather than dots).

3-level lists in rst get a little wonky, but 2-level lists should be kept as ol.

Reply:
ok! in that case there are few left over * just below this item